src/cljam/util/whole_genome.clj

```diff
@@ -0,0 +1,53 @@
+(ns cljam.util.whole-genome
+  "Utilities for conversions between chromosomal positions and whole-genome positions.")
+
+(defn create-chr-to-whole-genome-index
```

Comment:
In cljam, `create-*` is used for file-generating functions such as `bam-indexer/create-index` and `dict/create-dict`. I think `make-*` is better for this function, or simply `chr-to-whole-genome-index` is also good.